release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -74,6 +74,14 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-networking"]
 === Networking
 
+[id="ocp-4-12-rsp-workload-hints"]
+==== Disabling realtime using workload hints removes Receive Packet Steering from the cluster 
+By default, a systemd service sets a Receive Packet Steering (RPS) mask for virtual network interfaces in the cluster. The RPS mask routes interrupt requests from virtual network interfaces according to the list of reserved CPUs defined in the performance profile. A crio hook script also sets an RPS mask for all virtual network devices at the container level.
+
+With this update, if you set `spec.workloadHints.realTime` in the performance profile to `False`, the system also disables both the systemd service, and the crio hook script, that provides RPS to the cluster. The system disables these RPS functions when you disable realtime because RPS is typically relevant to use cases requiring low-latency, realtime workloads only.
```

Comment:
that provides RPS to the cluster -> that sets RPS mask according to reserved CPUs set in the performance profile